components/LinkGroup/src/index.scss

```diff
@@ -10,28 +10,32 @@
   margin-block-end: var(--denhaag-link-group-caption-margin-block);
 }
 
-.denhaag-link-group__navigation-list {
-  margin-block-start: var(--denhaag-link-group-navigation-list-margin-block-start);
-  margin-block-end: var(--denhaag-link-group-navigation-list-margin-block-end);
-  list-style: var(--denhaag-link-group-navigation-list-list-style);
-  padding-inline-start: var(--denhaag-link-group-navigation-list-padding-inline-start);
+.denhaag-link-group__list {
+  margin-block-start: var(--denhaag-link-group-list-margin-block-start);
+  margin-block-end: var(--denhaag-link-group-list-margin-block-end);
+  list-style: var(--denhaag-link-group-list-list-style);
+  padding-inline-start: var(--denhaag-link-group-list-padding-inline-start);
 
   @media (min-width: 1114px) {
     margin-block-start: var(--denhaag-space-block-sm);
   }
 }
 
-.denhaag-link-group__navigation-list-item {
-  padding-block-start: var(--denhaag-link-group-navigation-list-item-padding-block);
-  padding-block-end: var(--denhaag-link-group-navigation-list-item-padding-block);
+.denhaag-link-group__list-item {
+  padding-block-start: var(--denhaag-link-group-list-item-padding-block);
+  padding-block-end: var(--denhaag-link-group-list-item-padding-block);
 }
 
 .denhaag-link-group--dark .denhaag-link,
 .denhaag-link-group--dark .denhaag-link:hover,
-.denhaag-link-group--dark .denhaag-link:focus {
+.denhaag-link-group--dark .denhaag-link:focus-visible {
   color: var(--denhaag-link-group-dark-denhaag-link-color);
 }
 
+.denhaag-link-group--dark .denhaag-link:focus-visible {
+  outline-color: var(--denhaag-link-group-dark-denhaag-link-focus-outline-color);
```

Comment:
🤔 waarom hebben sommige tokens 2x denhaag er in? Is niet nieuw in deze pr, maar denk dat het eigenlijk `--denhaag-link-group-dark-link-focus-outline-color` of zoiets zou moeten zijn.

Reply:
Deze heb ik over 't hoofd gezien, maar inderdaad de suggestie voor deze benaming is wel beter.

Reply:
> https://www.chromatic.com/pullrequest?activeElementId=comparison-test-key-62d52cda68f01acbab1474bb-1200&appId=62d1558ac36c853a5aaad43f&number=1030&view=changes
> 
> Ziet er ook uit alsof deze change een visuele verandering meebrengt, is het de bedoeling dat er nu minder whitespace rondom de highlighted links zit?



> https://www.chromatic.com/pullrequest?activeElementId=comparison-test-key-62d52cda68f01acbab1474bb-1200&appId=62d1558ac36c853a5aaad43f&number=1030&view=changes
> 
> Ziet er ook uit alsof deze change een visuele verandering meebrengt, is het de bedoeling dat er nu minder whitespace rondom de highlighted links zit?

@Yolijn , er waren wat styling correcties hiervoor doorgevoerd, heb je goed gezien.